release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -82,9 +82,60 @@ This release adds improvements related to the following components and concepts.
 === Machine API
 
 
+[id="ocp-4-8-mco-upgrade-complete"]
+==== Updates now report status based on all machine config pools
+
+When updating, the Machine Config Operator (MCO) now waits for the worker machine config pools (MCP) to finish updating before rpeorting that the update is complete. Previously, the MCO reported update progress on the status of the master machine config pools, even if the worker pools were not done updating.   
```

Comment:
nit: "rpeorting" -> "reporting"

Reply:
Also, I may be misunderstanding, but if this is documenting [this code change][1], then the MCO will complete the current update after required pools are updated (as it did in 4.7).  The new-in-4.8 change is that the MCO sets Upgradeable=False as long as there are any incomplete pools.  That blocks *future* minor updates, but does not block future patch updates, or the current update (if any).  Which is nice, because compute updates with lots of nodes and slow-to-drain workloads can take a while, and the current MCO behavior keeps it easy to pick up control-plane CVE fixes and the like in the meantime.

[1]: https://github.com/openshift/machine-config-operator/pull/2487#issue-599945083